lib/foreman_maintain/repository_manager/el.rb

```diff
@@ -76,10 +76,21 @@ def enabled_repos
     end
 
     def hash_of_repoids_urls(repos, regex)
-      Hash[*repos.split("\n").grep(regex).map do |entry|
-             entry.split(':', 2).last.strip
-           end
+      ids_urls = Hash[*repos.split("\n").grep(regex).map do |entry|
+                        entry.split(':', 2).last.strip
+                      end
       ]
+
+      if el7?
```

Comment:
given it doesn't hurt on EL8 either, why not always execute this, removing the branching complexity here?

(we can still have a comment a la `# remove anything after the first slash in the url, like EL7 returns it`